spec/index.html

```diff
@@ -1075,6 +1081,23 @@ <h4>Patterns of RDF entailment (Informative)</h4>
           </tr>
         </tbody>
       </table>
+ -->
+ 
+       <table>
+        <tbody>
+          <tr>
+            <th></th>
+            <th><strong>if the triple or triple term <a>appears in</a> S</strong></th>
```

Comment:
```suggestion
            <th><strong>if the triple <a>appears in</a> S</strong></th>
```
"or triple term" is redundant. The definition of "appears in" for triples covers it.

Reply:
I'm not opposed to keeping it, but I slightly prefer to remove it.

Reply:
I believe it would be wrong to assume that the check should be done only for triples.

Reply:
@pchampin : do you agree?

Reply:
> I believe it would be wrong to assume that the check should be done only for triples.

It would indeed be wrong to assume that,
because the definition of "a triple appears in a graph" explicitly says that triple *terms* must be checked as well.

That's why I consider that "or triple term" in this sentence (and its twin below) is not needed.
But as I wrote, I'm not strongly opposed to keeping it. It is not wrong, just redundant.

Reply:
Ops, you are right, I have reread the definition. Will fix it.